BoxContentSDK/BoxContentSDK/External/KeychainItemWrapper/BOXKeychainItemWrapper.m

```diff
@@ -400,4 +395,41 @@ - (NSMutableDictionary *)defaultKeychainItemDataDictionaryWithIdentifier:(NSStri
     return defaultDictionary;
 }
 
+// Retrieve all keychain entries that have been made by instances of BOXKeychainItemWrapper.
+// Sample usage in arc land: NSArray* keychainEntries = (__bridge_transfer NSArray*)[BOXKeychainItemWrapper allEntries];
```

Comment:
Whoops, this was a comment for an older version of this method. I added the correct comment to the .h. Will remove this.